### PR TITLE
add tests

### PR DESCRIPTION
**Pull request by ghost**
Note: I had to run this on master, since I ran into some issues with these additions:
https://github.com/carnivore-rb/jackal-github-kit/blob/develop/lib/jackal-github-kit.rb#L11
https://github.com/carnivore-rb/jackal/blob/develop/lib/jackal.rb#L38

Initially:
`undefined method `service' for Jackal:Module (NoMethodError)`
And after pointing gemfile at develop branch for jackal:
`<internal:prelude>:11:in `synchronize': deadlock; recursive locking (ThreadError)
from <internal:prelude>:11:in `exclusive'
from /home/slackz/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/bogo-config-0.1.10/lib/bogo-config/config.rb:126:in `init!'
from /home/slackz/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/bogo-config-0.1.10/lib/bogo-config/config.rb:96:in `initialize'
from /home/slackz/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/bundler/gems/jackal-bf1d6dddbbd5/lib/jackal.rb:25:in `new'`

### CODE REVIEW

**Comment by ghost** [comment]
Also, this requires a dummy github account (for use of the access_token).  This account will need create/delete access for repos.

File: jackal-github-kit.gemspec [hunk 1]
```diff
@@ -12,5 +12,6 @@ Gem::Specification.new do |s|
   s.license = 'Apache 2.0'
   s.add_dependency 'jackal'
   s.add_dependency 'octokit'
+  s.add_development_dependency 'carnivore-actor'
```
Comment (chrisroberts) [inline jackal-github-kit.gemspec:15]: This should be in the Gemfile so it's provided via the bundle for testing

Comment (chrisroberts) [inline jackal-github-kit.gemspec:15]: Disregard me

**Comment by chrisroberts** [comment]
:+1:

**Pull request merged**